screen-editor.tex

```diff
@@ -1,44 +1,40 @@
 \section{The Screen Editor}
 \label{sec:screen-editor}
 
-When your turn on your MEGA65, or reset it, the editor screen will appear.
+When your turn on your MEGA65 or reset it, the following screen will appear:
 
 \begin{center}
 \includegraphics[width={10cm}]{images/introduction-screen/layout.png}
 \end{center}
 
-The colour bars in the top left hand of the screen can be used as a guide to help calibrate the colours of your display. The screen also displays the name of the system, the copyright notice and what version and revision of BASIC is contained in the Read-only Memory.
+The colour bars in the top left hand of the screen can be used as a guide to help calibrate the colours of your display. The screen also displays the name of the system, the copyright notice, and the version of BASIC that is contained in Read-only memory.
 
-Also displayed is the type of keyboard and whether or not there is additional hardware present, such as a RAM expansion.
+Also displayed is the type of keyboard, and whether there is any additional hardware present, such as a RAM expansion.
 
 Finally, you will see the READY prompt and the flashing cursor.
 
-You can begin typing keys on the keyboard and the characters will be printed under the cursor. The cursor itself advances after each key press.
+You can begin typing keys on the keyboard and the characters will be printed at the cursor position. The cursor itself will advance after each key press.
 
-You can also produce reverse text or colour bars by holding down the \megakey{CTRL} key and pressing the \megakey{9} key or the \megakey{R} key. This enters reverse text mode.
+You can also produce reverse text or colour bars by holding down the \megakey{CTRL} key and pressing the \megakey{9} key or the \megakey{0} key. These combinations enable and disable  the reverse text mode.
```

Comment:
I'm thinking the original paragraph is more appropriate as the CTRL+R key combo does indeed also turn on reverse text, and the explanation of using CTRL+0 to turn off reverse text is given in a later paragraph below.